aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -244,6 +244,8 @@ def define(cls, spec: CalcJobProcessSpec) -> None:  # type: ignore[override]
             help='If set to true, the submission script will load the system environment variables',)
         spec.input('metadata.options.environment_variables', valid_type=dict, default=lambda: {},
             help='Set a dictionary of custom environment variables for this calculation',)
+        spec.input('metadata.options.use_double_quotes', valid_type=bool, default=False,
```

Comment:
Since this only applies to the `environment_variables` option, should we name this option more explicitly, like `environment_variables_double_quotes`? This makes it a lot clearer that it only affects that.

Reply:
Maybe `envvar_double_quotes` instead of `environment_variables` to avoid to have a super-long name that users have to type? I'm not 100% sure of this name though

Reply:
I agree it is not precise to naming it as `use_double_quotes` but `environment_variables_double_quotes` really too long. My initial consideration is that `use_double_quotes` also influent other parts like command line parameters. Do you think it is too much to do this in the calcjob metadata setting with one option?